muspinsim_input/muspinsim_input.xml

```diff
@@ -102,69 +108,75 @@
-                            ">
+                <param name="experiment_preset" type="select" value='custom' display="radio" label="Experiment type"
+                       help="Avoided Level Crossing (ALC) - polarization: longitudinal, x-axis: field, y-axis: integral
+                             Zero field - polarization: transverse, field: 0
+                             Custom - No preset">
+
                     <option value="alc">Avoided Level Crossing (ALC)</option>
                     <option value="zero_field">Zero Field</option>
                     <option selected="true" value="custom">Custom</option>
                 </param>
                 <when value="alc">
-                    <repeat name="fields" title="Magnetic fields to simulate">
-                        <param name="field" type="text" value="0" label="Enter field to simulate, either scalar of vector"
+                    <repeat name="fields" title="Fields" help="Magnetic fields">
+                        <param name="field" type="text" value="0" label="Field"
                             optional="true" help="Optional, (default is 0).
                             Accepts 1X3 vector, or scalar value. Scalar value means field is assumed to be aligned with z axis.
-                            Allows default expressions, constants and functions.
-                            Including MHz, muon_gyr constants and special 'range()' function (see help section)"/>
+                            Allows default expressions, constants and functions, plus MHz, muon_gyr constants and special 'range()' function (see help section)"/>
                     </repeat>
                 </when>
                 <when value="zero_field">
-                    <param name="x_axis" type="select" value="time" display="radio" label="Choose which range to use as x axis for output file(s)">
+                    <param name="x_axis" type="select" value="time" display="radio" label="X axis"
+                           help="Range to use as X axis for output file(s)">
                         <option selected="true" value="time">time</option>
                         <option value="field">field</option>
                     </param>
-                    <param name="y_axis" type="select" value="asymmetry" display="radio" label="Choose which range to use as y axis for output file(s)"
-                           help="Asymmetry: Use muon's polarization, Integral: use muon's polarization integral over time,
+                    <param name="y_axis" type="select" value="asymmetry" display="radio" label="Y axis"
+                           help="Range to use as y axis for output file(s)
+                                Asymmetry: Use muon's polarization,
+                                Integral: use muon's polarization integral over time,
                                 WARNING: if integral chosen, 'time' parameter is ignored, and cannot be used as x-axis parameter ">
```

Comment:
Same comment here about line endings, for the first line in particular. This formatting should be consistent with the Experiment Type hint text - notice you've used colons in two slightly different ways, and that can be confusing.

Reply:
Ok, I actually prefer the way you had it before. I'd just put full stops between the lines instead of commas as the only change